<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JustListen</class>
 <widget class="QMainWindow" name="JustListen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JustListen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="connectionBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>231</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QLabel" name="labelIp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>36</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ip:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineIp">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.29.3</string>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelPorta">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>67</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonConect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>90</width>
       <height>42</height>
      </rect>
     </property>
     <property name="text">
      <string>Conect to
OpenServer</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="boxStatus">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonDisconect">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconect</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="boxPort">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>161</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="value">
      <number>54000</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="speedBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>211</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Speed:</string>
     </property>
     <widget class="QRadioButton" name="radioButton_1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>400 us</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>2 ms</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>4 ms</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>8 ms</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>16 ms</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>481</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Joins</string>
    </property>
    <widget class="QSlider" name="SJ1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-170</number>
     </property>
     <property name="maximum">
      <number>170</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSlider" name="SJ2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-85000</number>
     </property>
     <property name="maximum">
      <number>155000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="SJ3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-170000</number>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="SJ4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-210000</number>
     </property>
     <property name="maximum">
      <number>210000</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="SJ6">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2700000</number>
     </property>
     <property name="maximum">
      <number>2700000</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="SJ5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-130000</number>
     </property>
     <property name="maximum">
      <number>130000</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ax1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ax2:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ax3:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ax4:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ax5:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ax6:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="TJ1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-170.000000000000000</double>
     </property>
     <property name="maximum">
      <double>170.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="TJ2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-85.000000000000000</double>
     </property>
     <property name="maximum">
      <double>155.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="TJ3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-170.000000000000000</double>
     </property>
     <property name="maximum">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="TJ4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-210.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="TJ5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-130.000000000000000</double>
     </property>
     <property name="maximum">
      <double>130.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="TJ6">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>320</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-270.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>10</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>150</y>
       <width>10</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>210</y>
       <width>10</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>270</y>
       <width>10</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>330</y>
       <width>10</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>JustListen</sender>
   <signal>j1(double)</signal>
   <receiver>TJ1</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JustListen</sender>
   <signal>j2(double)</signal>
   <receiver>TJ2</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JustListen</sender>
   <signal>j3(double)</signal>
   <receiver>TJ3</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JustListen</sender>
   <signal>j4(double)</signal>
   <receiver>TJ4</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JustListen</sender>
   <signal>j5(double)</signal>
   <receiver>TJ5</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JustListen</sender>
   <signal>j6(double)</signal>
   <receiver>TJ6</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JustListen</sender>
   <signal>j1_int(int)</signal>
   <receiver>SJ1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JustListen</sender>
   <signal>j6_int(int)</signal>
   <receiver>SJ6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JustListen</sender>
   <signal>j5_int(int)</signal>
   <receiver>SJ5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JustListen</sender>
   <signal>j4_int(int)</signal>
   <receiver>SJ4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JustListen</sender>
   <signal>j3_int(int)</signal>
   <receiver>SJ3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JustListen</sender>
   <signal>j2_int(int)</signal>
   <receiver>SJ2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>j1(double)</signal>
  <signal>j2(double)</signal>
  <signal>j3(double)</signal>
  <signal>j4(double)</signal>
  <signal>j5(double)</signal>
  <signal>j6(double)</signal>
  <signal>j1_int(int)</signal>
  <signal>j2_int(int)</signal>
  <signal>j3_int(int)</signal>
  <signal>j4_int(int)</signal>
  <signal>j5_int(int)</signal>
  <signal>j6_int(int)</signal>
 </slots>
</ui>
